js/main.js

```diff
@@ -0,0 +1,59 @@
+const myBookList = [];
+const submit = document.querySelector('#submit');
+const list = document.querySelector('#bookList');
+const bookTitle = document.querySelector('#inputTitle');
+const bookAuthor = document.querySelector('#inputAuthor');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
```

Comment:
- [ ] Adding/removing a book from the library, books should be preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), I'd suggest also using the `window.onload` function to check if there is an existing booklist in the localStorage.

Reply:
local Storage is currently included in the new request 👏